app/views/layouts/spotlight/base.html.erb

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<%= content_tag :html, class: 'no-js', **(try(:html_tag_attributes) || {}) do %>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <%= render 'shared/analytics' %>
+    <title><%= h(@page_title || application_name.to_s) %></title>
+    <link href="<%= current_exhibit ? spotlight.opensearch_exhibit_catalog_url(current_exhibit, format: 'xml') : main_app.opensearch_catalog_url(format: 'xml') %>" title="<%= h(@page_title || application_name.to_str) %>" type="application/opensearchdescription+xml" rel="search"/>
+    <%= favicon_link_tag 'favicon.ico' %>
+    <% if current_exhibit %>
+      <%= exhibit_stylesheet_link_tag "application" %>
+    <% else %>
+      <%= stylesheet_link_tag "application" %>
+    <% end %>
+    <%= javascript_include_tag "application" %>
+    <%= csrf_meta_tags %>
+    <%= content_for(:head) %>
+    <%= description %>
+    <%= twitter_card %>
+    <%= opengraph %>
+    <%= javascript_tag '$.fx.off = true;' if Rails.env.test? %>
+  </head>
+  <body class="<%= render_body_class %>">
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-T96MPMB" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
@cbeer What's this GTM ID?

Reply:
@cbeer oh, nvm, that's the production ID. I don't think we'd want to hardcode that here, but maybe I'm mistaken.